Cabal/src/Language/Haskell/Extension.hs

```diff
@@ -849,6 +849,18 @@ data KnownExtension =
   -- | Enable linear types.
   | LinearTypes
 
+  -- | Disable the generation of selector functions corresponding to record fields.
+  | NoFieldSelectors
+
+  -- | Enable the use of record dot-accessor and updater syntax
+  | RecordDotSyntax
+
+  -- | Enable the use of the GHC2021 collection of language extensions.
+  | GHC2021
```

Comment:
This is not `Extension`. This is `Language`.

Reply:
Shoot my bad - I was thinking extension for some reasno.